app/kuma-dp/pkg/dataplane/envoy/envoy.go

```diff
@@ -96,7 +103,13 @@ func lookupEnvoyPath(configuredPath string) (string, error) {
 
 func (e *Envoy) Start(stop <-chan struct{}) error {
 	runLog.Info("generating bootstrap configuration")
-	bootstrapConfig, err := e.opts.Generator(e.opts.Config.ControlPlane.URL, e.opts.Config, e.opts.Dataplane)
+	envoyVersion, err := e.version()
+	if err != nil {
+		return errors.Wrap(err, "failed to get Envoy version")
+	}
+	runLog.Info("fetched Envoy version", "version", envoyVersion)
+	runLog.Info("generating bootstrap configuration")
```

Comment:
you forgot to remove the same log in 105

Reply:
Ok, I removed this redundant log.